README.md

```diff
@@ -1,223 +1,209 @@
 
-#### Image directory
+All of the parameters defining the Metashape workflow are specified in the configuration file (a [YAML-format](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html) file). This includes directories of input and output files, workflow steps to include, quality settings, and many other parameters.
 
-The images to be processed should all be in one folder (and optionally organized into subfolders beneath this folder) somewhere within the data folder you will be mounting. In the example above, the folder of images is at `~/drone_data/projects/project_10/images/`.
+An example configuration file is provided in this repo at `config/config-base.yml`. Edit the parameter values to meet your specifications. The file contains comments explaining the purpose of each customizable parameter.  You can directly edit the `config-base.yml` or save a new copy somewhere on the your local computer. You will specify the path of this config.yml in the [python run command](#running-the-workflow).
 
-#### Workflow configuration file
+Note: Please do not remove or add parameters to the configuration file; adding will have no effect unless the Python code is changed along with the addition, and removing will produce errors.
 
-An example configuration file is provided in this repository at `config/config-example.yml`. Please download this file to your local machine and rename it `config.yml`. By default (but see below), the container expects the config YAML file describing the Metashape workflow parameters to be located at `/data/config.yaml`. So in a case where the local folder to be mounted is `~/drone_data/`, then for the config file to be mounted in the Docker container at `/data/config.yaml`, it must be located on the local computer at `~/drone_data/config.yaml`. However, the config file location can be overridden by passing a different location following an optional command line argument `--config_file` of the `docker run` command (see below).
+<br/>
 
-Within the `config.yml` you will need to edit some of the project level parameters to specify where to find input images and where to put output products within the container. Within this config file, all paths will be relative to the file structure of the docker container (beginning with `/data/`). In the config.yaml, at a minimum the following entries should be updated:
+### Running the Workflow
 
-* The value for photo_path should be updated to /data/{path_to_images_folder_within_mounted_folder}
-* The value for output_path should be updated to /data/{path_to_desired_ouputs_folder_within_mounted_folder} (can be any location you want; will be created if it does not exist)
-* The value for project_path should be updated similarly as for output_path.
+The general command line call to run the worflow has three components:
+1. Call to Python
+2. Path to metashape workflow Python script (`metashape_workflow.py`)
+3. Path to workflow configuration file (`*.yml`)
```

Comment:
This is slightly more complex now because there's the option to override common parameters on the command line, which means the default config could actually be used in some cases.

Reply:
Yes, that's correct. The default path is computed [here](https://github.com/open-forest-observatory/automate-metashape/blob/b8059c28a36abb1c51d264d3e895531d7b2b9b4e/python/metashape_workflow.py#L13).